solutions/observability/apm/resource-attributes.md

```diff
@@ -36,13 +40,116 @@ processors:
+| `exception.type`, `exception.message`, `exception.stacktrace`, `exception.escaped` | `Error.Exception`                         | Maps exception attributes to the ECS error exception fields (type, message, and stacktrace).                                                                                                                   |
+| `elastic.profiler_stack_trace_ids`                 | `Transaction.ProfilerStackTraceIds`       | Converts the profiler stack trace IDs to a list.                                                                                                                                                                                                      |
+| `http.user_agent`, `user_agent.original`           | `UserAgent.Original`                      | Maps the user agent string directly to the ECS field.                                                                                                                                                                                                 |
+| `network.connection.type`, `network.connection.subtype`, `network.carrier.*` | `Network.Connection`, `Network.Carrier`   | Maps the network connection type, subtype, and carrier details to the ECS fields.                                                                                                                                                                     |
+| `span.kind`                                        | `Transaction.Type` or `Span.Type`         | Determines the transaction or span type based on the span's kind. For example, `SpanKindConsumer` results in a transaction type of `messaging`.                                                                 |
+| `peer.service`                                     | `Destination.Service.Name`                | Maps the peer service name to the destination service name.                                                                                                                                                                                            |
+| `message_bus.destination`, `messaging.temp_destination` | `Destination.Service.Resource`            | Appends the message bus destination or queue name to the service resource.                                                                                                                                                                             |
+| `elastic.is_child`, `is_child`                     | `Span.ChildIds`                           | Determines if a span link is a child span and populates the `ChildIds` field accordingly.                                                                                                                                                              |
+
+Consider the following resource attributes:
+
+```json
+{
+  "http.status_code": 200,
+  "feature.enabled": true,
+  "http.request_headers": ["accept:json", "auth:token"]
```

Comment:
As discussed, this example is not spec compliant: https://opentelemetry.io/docs/specs/semconv/attributes-registry/http/#http-attributes
Removing to avoid confusion
```suggestion
  "feature.enabled": true
```